examples/example_resources/lib/gen/assets.gen.dart

```diff
@@ -3,18 +3,18 @@
 ///  FlutterGen
 /// *****************************************************
 
+import 'package:flare_flutter/flare_actor.dart';
+import 'package:flare_flutter/flare_controller.dart';
+import 'package:flutter/services.dart';
```

Comment:
Why these lines are changed?

Reply:
I'm not sure, it must have formatted it like that. I've reverted all the formatting/generated changes.